api/api/models/audio.py

```diff
@@ -192,7 +194,16 @@ class Audio(AudioFileMixin, AbstractMedia):
     alt_files = models.JSONField(
         blank=True,
         null=True,
-        help_text="JSON describing alternative files for this audio.",
+        help_text=d("""
+        JSON object containing information on alternative audio files. Each object
+        is expected to contain:
+
+        - `url`: URL reference to the file
+        - `filesize`: File size in bytes
+        - `filetype`: Extension of the file
+        - `bit_rate`: Bitrate of the file in bits/second
+        - `sample_rate`: Sample rate of the file in bits/second
```

Comment:
Part of the reason for documenting this here is that it will show up in the [API Media Properties page](https://docs.openverse.org/meta/media_properties/api.html). Since this is on the model and not the serializer, I'm not sure it'd show up on the OpenAPI documentation. I'll have to do some experimentation with that.

Reply:
Oops, I totally missed that! You're right. I still don't think it should be documented here and that it should be a reference to some shared Tag documentation, but ignore the part I said about the OpenAPI documentation. It won't have any side-effect there.